src/Framework/MockObject/Generator.php

```diff
@@ -249,6 +249,8 @@ interface_exists($originalClassName, $callAutoload)) {
      * of the trait mocked. Concrete methods to mock can be specified with the
      * `$mockedMethods` parameter.
      *
+     * @param trait-string $traitName
```

Comment:
For Psalm-specific annotations we use `@psalm-param`, at least so far. I think for this pull request it would be best to stick with that. But we should think about moving from `@psalm-param`, `@psalm-return`, and `@psalm-var` to `@param`, `@return`, and `@var`. How do other projects handle this? Pinging @Ocramius and @muglug out of the blue (sorry!).

Reply:
Yep, using `@psalm-param` would be best to not annoy other static analysis tools that don't recognize this format.

What I generally do is combining `@psalm-param` for the actual type (if complex enough that only psalm can understand it) plus `@param` with a simpler type that is, for example, IDE friendly.

Reply:
I think this patch is fine as-is, @sebastianbergmann

Reply:
@sebastianbergmann 

Adjusted!

I remember @Ocramius mentioning that he doesn't use the`@psalm-` prefix.

Reply:
I do, but I am also trying to push the ecosystem forward. Moving from `@psalm-` to unprefixed will crash some tools somewhere, so be careful 😬

Reply:
As long as it does not crash any tool we are using ... 😬